server/server.js

```diff
@@ -220,11 +234,22 @@ Server.prototype._handleResourceMessage = function(socket, message, messageType)
     this._storeResource(resource);
     this._persistenceSubscribe(resource.name, socket.id);
     this._updateLimits(socket, message, resource.options);
+    this._stampMessage(socket, message);
```

Comment:
I'll spend some time splitting this PR, and will deprecated the current one. It makes sense.